src/main/java/org/folio/circulation/resources/RequestByInstanceIdResource.java

```diff
@@ -259,19 +259,20 @@ private void createInstanceLevelRequests(RoutingContext routingContext) {
     }
 
     JsonObject currentItemRequest = itemRequests.get(startIndex);
-
+    ItemRepository itemRepository = new ItemRepository(clients, true, false, false);
```

Comment:
using multiple boolean arguments does not look smell, would prefer enums

Reply:
what do you mean by preferring enums?